tests/testthat/test-auditclassicalbenfordslaw.R

```diff
@@ -1,13 +1,13 @@
 context("[Audit] Benford's Law")
 
-options <- jaspTools::analysisOptions("auditClassicalBenfordsLaw")
+options <- analysisOptions("auditClassicalBenfordsLaw")
```

Comment:
Nitpick: if you keep the `jaspTools::` then you don't need `library(jaspTools)` to test the package.